### PR TITLE
Added specific code-along references

### PR DESCRIPTION
**Pull request by payne-chris-r**
Added "CODE ALONG" and some specific references to allow the developers some hands on practice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,124 +6,151 @@
...
 
 The ES6 primitive type `Symbol` is intentionally omitted.
 
 ### Literals
 
-Literals represent specific values in the source code.  Some examples are `1`, `'A string'`, `null`.
+Literals represent specific values in the source code.
+Some examples are `1`, `'A string'`, `null`.
+
+## CODE ALONG
```
Comment (jrhorn424) [inline README.md]: Please use headings like "Code-Along: Imperative Verb a Something Describing Task"

**Comment by jrhorn424** [comment]
@payne-chris-r Please address my comment and I'll merge. Thanks!

**Comment by payne-chris-r** [comment]
Addressed.

File: README.md [hunk 2]
```diff
@@ -6,124 +6,151 @@
...
+
+We'll use Node.js as a [REPL](https://nodejs.org/api/repl.html) and script
+runner to evaluate expressions and explore JavaScript features.
+
+-   **R**ead
+-   **E**valuate
+-   **P**rint
+-   **L**oop
+
+#### Code Along: Intro into Node and Variables
```
Comment (jrhorn424) [inline README.md]: Better: `Code-Along: Declare Variables`

Comment (payne-chris-r) [inline README.md]: Done @jrhorn424

**Comment by gaand** [comment]
Although `dob` should have been called `bornOn` I prefer it to `name` as it served the pedagogical purpose of illuminating ISO 8601 dates.

**Pull request merged**